docs-chef-io/content/inspec/resources/k8s_job.md

```diff
@@ -12,54 +12,54 @@ parent = "inspec/resources/k8s"
 +++
 
 
-Use the `k8s_job` Chef InSpec audit resource to test the configuration of a specific Job in the specified namespace.
+Use the `k8s_job` Chef InSpec audit resource to test the configuration of a specific job in the specified namespace.
```

Comment:
Why is Job changed to job?
Similarly, Job and Jobs are types of Kubernetes resource.